doc/overrides.css

```diff
@@ -1,22 +0,0 @@
-.docbook .xref img[src^=images\/callouts\/],
-.screen img,
-.programlisting img,
-.literallayout img,
-.synopsis img {
-    width: 1em;
-}
-
-.calloutlist img {
-    width: 1.5em;
-}
-
-.prompt,
```

Comment:
This regressed the prompts yet again. Previously:

- https://github.com/NixOS/nixpkgs/pull/77683
- https://github.com/NixOS/nixos-homepage/issues/1094
- https://github.com/NixOS/nixos-homepage/pull/492

Reply:
Prompts were broken before this change. See e.g. [this snapshot](https://web.archive.org/web/20240124234518/https://nixos.org/manual/nixpkgs/unstable/) from Jan 24.

Having said that, highlight.js seems to add [certain classes](https://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#stylable-scopes) that will help make prompts unselectable again. I'll work on a fix for this (likely tomorrow).

Reply:
Hmm, appears they only worked on the website itself: https://web.archive.org/web/20240125001545/https://nixos.org/download

Reply:
https://github.com/NixOS/nixpkgs/pull/301439